src/service/models.py

```diff
@@ -294,6 +295,8 @@ def get_data_product(self, data_product) -> DataProduct:
         """ Get a data product by its ID. """
         for dp in self.data_products:
             if dp.product == data_product:
+                if generic_view.is_generic_view_product(data_product):
+                    dp.search_view = generic_view.get_generic_view_name(data_product)
```

Comment:
There's no reason to set a generic search view on a collection, since they're always the same for all collections.

Reply:
👍

Reply:
:+1: